posthog-react-native/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Next
 
+# 3.6.0 - 2024-12-04
+
+1. Add new debugging property `$feature_flag_bootstrapped_response`, `$feature_flag_bootstrapped_payload` and `$used_bootstrap_value` to `$feature_flag_called` event
+
 # 3.5.0 - 2024-12-03
 
 1. fix: deprecate maskPhotoLibraryImages due to unintended masking issues
 
 # 3.4.0 - 2024-11-26
 
 1. feat: automatically mask out user photos and sandboxed views like photo picker (iOS Only)
-  1. To disable masking set `maskAllSandboxedViews` and `maskPhotoLibraryImages` to false
+1. To disable masking set `maskAllSandboxedViews` and `maskPhotoLibraryImages` to false
```

Comment:
I believe those changes (the rest of the file) are unintentional since its a nested bullet list, please revert

Reply:
Weird my VS code was doing this, reverted